1-mandatory/1-writers.js

```diff
@@ -61,8 +66,20 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
+function getDeadWritersInTheirForties(writer) {
+  return writer.age > 40 && writer.age < 49 && writer.alive === false;
+}
+
+function printSentence(writer) {
+  return console.log(
+    `Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`
+  );
+}
+
 function logDeadWritersInTheirForties() {
-  // write your code here
+  let deadWritersInTheirForties = writers.filter(getDeadWritersInTheirForties);
+
+  deadWritersInTheirForties.map(printSentence);
```

Comment:
Issue: `map` isn't appropriate here as we're not modifying the items in the array. I think a `foreach` would be appropriate as we're taking an action on every item.